### ISSUE TITLE
v4.0.1: Error: Nativewind: was unable to detect the 'nativewind/preset'

### ISSUE THREAD

**Issue by fendyk** [marklawlor/nativewind#603]
**Describe the bug**
When using the latest version and doing all the steps required for Nativewind (with expo-router) you get this error. also, it shows this in the console:

`[{"property": "margin-left", "type": "IncompatibleNativeValue", "value": "auto"}, {"property": "margin-right", "type": "IncompatibleNativeValue", "value": "auto"}]`

**NOTE**: All styles work for some reason. It's just that when I use the `verifyInstallation()` function it displays the error

**To Reproduce**
1. Go to https://www.nativewind.dev/v4/getting-started/expo-router
2. Follow al the steps in a blank expo-router typescript app (SDK49)
3. Use `verifyInstallation();`

**Expected behavior**
Should work as intended, since I've done all the steps required. 

My `tailwindcss.config.js`:
```javascript
/** @type {import('tailwindcss').Config} */
module.exports = {
    content: [
        './app/**/*.{js,jsx,ts,tsx}',
    ],
    presets: [require("nativewind/preset")],
    theme: {
        extend: {},
    },
    plugins: []
}

```

My `babel.config.js`:
```javascript
module.exports = function (api) {
    api.cache(true);
    return {
        presets: [
            ["babel-preset-expo", {jsxImportSource: "nativewind"}],
            "nativewind/babel",
        ],
        plugins: [
            // Required for expo-router
            'expo-router/babel',
            'react-native-reanimated/plugin'
        ],
    };
};
```

My `metro.config.js`:
```javascript
const { getDefaultConfig } = require("expo/metro-config");
const { withNativeWind } = require('nativewind/metro');

const config = getDefaultConfig(__dirname, { isCSSEnabled: true })

module.exports = withNativeWind(config, { input: './global.css' })
```

**Comment by robcaldecott** [comment]
Same issue here with an Expo 49 app (not Expo Router). Many of my styles are broken (`flex-row` is not working for some readon) and auto refresh is not working.

**Comment by marklawlor** [comment]
@robcaldecott Your issue seems very different to the one being reported (in which they say the styles are working, and it's a false positive on the verify helper). Please create a new issue with a link to your repo so I can investigate.

**Comment by robcaldecott** [comment]
@marklawlor here you go:

https://github.com/robcaldecott/nativewind-4-issues

It's an Expo app and works with Expo Go.

Issues:

- Refresh when changing any code is not working in iOS or Android simulators. It sometimes struggles to connect at all and requires Expo Go in each simulator to be killed off.
- Not all (if any) `Pressable` styles are being applied (`active:`, background, border, etc. are not working).
- The `View` with the Google logo and `Text` seems to ignore `flex-row`. This is code lifted from our actual app.

I have kept the repo with the minimum of dependencies but have included SVG support. It's definitely applying some Tailwind styles so I think it's close.

**Comment by marklawlor** [comment]
Fixed in `4.0.8`
